README.md

```diff
@@ -1 +1 @@
-# mslice
\ No newline at end of file
+# MSlice3.0
```

Comment:
Can we have some more information in this. It doesn't need to contain too much. But it should include 

1. A statement about the fact that this is a prototype, and currently not production ready. 
1. What is this application supposed to do. Only need to give one or two sentences for this at a very high level.
1. Information on what modules are needed to run this. The fact that MantidPlot needs to be installed etc. Which version of MantidPlot. 
1. How to run the tests
1. How to run the `QuickView`
1. What existing projects this is based upon (MSlice 2.0, MSlice, etc)

Here are some good examples of readmes [1](https://github.com/DMSC-Instrument-Data/plankton), [2](https://github.com/DMSC-Instrument-Data/instrument-prototype)

You can make a separate Issue/PR for this if you wish.